src/lib/screens/explore.dart

```diff
@@ -1,86 +1,102 @@
         actions: <Widget>[
-          IconButton(icon: Icon(Icons.search), onPressed: () => {
-            showSearch(context: context, delegate: ConferenceSearch())
-          })
+          IconButton(
+              icon: Icon(Icons.search),
+              onPressed: () =>
+                  {showSearch(context: context, delegate: ConferenceSearch())})
         ],
+        backgroundColor: accentOrange,
       ),
-      drawer: Drawer(),
     );
   }
 }
 
 class ConferenceSearch extends SearchDelegate<String> {
-  final _allConferences = ["HELLO","MEGA CONFERENCIA", "WE LOVE YOU_ CONF"];
-  final _lastConferences = ["HELLO"];
+  // If the conference has image return the url, otherwise returns a default.
+  // Mudar para ficar mais modular!
+  hasImage(conference) {
+    return conference['img'] == null
+        ? 'http://www.theides.org/img/about.jpg'
+        : conference['img'];
+  }
 
   @override
   List<Widget> buildActions(BuildContext context) {
-    return [IconButton(icon: Icon(Icons.clear), onPressed: () {
-      query = "";
-    })];
+    return [
+      IconButton(
+          icon: Icon(Icons.clear),
+          onPressed: () {
+            query = "";
+          })
+    ];
   }
 
   @override
   Widget buildLeading(BuildContext context) {
-    return IconButton(icon: AnimatedIcon(
-      icon: AnimatedIcons.menu_arrow,
-      progress: transitionAnimation,
-    ), onPressed: () {
-      close(context, null);
-    });
+    return IconButton(
+        icon: AnimatedIcon(
+          icon: AnimatedIcons.menu_arrow,
+          progress: transitionAnimation,
+        ),
+        onPressed: () {
+          close(context, null);
+        });
   }
 
+
   @override
   Widget buildResults(BuildContext context) {
-    // TODO: implement buildResults
-    throw UnimplementedError();
-  }
-/*
-  @override
-  Widget buildSuggestions(BuildContext context) {
-    final suggestionList = query.isEmpty ?
-      _lastConferences:
-      _allConferences.where((p) => p.startsWith(query)).toList();
-    return ListView.builder(
-        itemBuilder: (context, index) => ListTile(
-          leading: Icon(Icons.location_city),
-          title: Text(suggestionList[index]),
-        ),
-      itemCount: suggestionList.length,
-    );
+
   }
-*/
+
   @override
   Widget buildSuggestions(BuildContext context) {
     return StreamBuilder(
       stream: FirebaseFirestore.instance.collection('Conference').snapshots(),
       builder: (context, snapshot) {
         if (!snapshot.hasData) return new Text('Loading...');
-        if(query == '') return new Text('');
-        final results = snapshot.data.documents.where((a) =>
-            a.data()['title'].toString().contains(query));
-       var b;
-       for (b in results){
-         print(b);
-       }
+        if (query == '') return new Text('');
+        final results = snapshot.data.documents
+            .where((a) => a.data()['title'].toString().contains(query));
```

Comment:
Try to ignore the case, using `.toLowerCase()`

Reply:
The search results can be not only related to the title, but also to other parameters, like tags, speakers names, location, etc.

Reply:
A parte do lower case já resolvi. A parte do pesquisar por outras coisas, não sei mt bem como seria para fazer. Seria várias barras de pesquisa, uma para cada tipo, ou ao pesquisar tudo e aparece tudo?

Reply:
Eu mudei. Ve como fiz agora

Reply:
Fazias tudo na mesma barrra de pesquisa, mas em vez de verificares o resultado da query com o atributo `título` da conferência, fazias match com mais atributos. No caso, ficaria algo do género:
```
final results = snapshot.data.documents
            .where((a) => (a.data()['title'].toString().contains(query)) || a.data()['location'].toString().contains(query)) || a.data()['tag'].toString().contains(query)) || a.data()['description'].toString().contains(query)));
```

Reply:
sim tens razão. Eu fiz uma função para evitar code duplicate. Vê como ficou